### ISSUE TITLE
mxml Error: "format"-Attribute

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Gamer125/rssmii#1]
```
What steps will reproduce the problem?
1. Add an RSS-Feed in the "feeds.xml" that is generated by a PHP-Script that 
takes a "format"-argument. ("http://example.com/feed.php?format=rss")


What is the expected output? What do you see instead?
rssmii should work completely normally, but instead, it tells you: "error -103"
```

Original issue reported on code.google.com by `Computer...@gmail.com` on 2 Jan 2011 at 12:21

**Comment by GoogleCodeExporter** [comment]
```
I think CDATA should solve the problem. I will create a branch and test that.
```

Original comment by `Computer...@gmail.com` on 2 Jan 2011 at 4:57
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Still working on that issue. It's planned to get fixed with Version 1.1 of 
rssmii
```

Original comment by `Computer...@gmail.com` on 3 Jan 2011 at 3:22
- Added labels: **Milestone-1.1**

**Comment by GoogleCodeExporter** [comment]
```
RSSMii 1.1 is now released and this issue is now fixed.
```

Original comment by `Computer...@gmail.com` on 9 Jan 2011 at 12:10
- Changed state: **Fixed**
